test/helpers/fixture_deploy_helper.rb

```diff
@@ -26,18 +26,24 @@ module FixtureDeployHelper
   #     pod = fixtures["unmanaged-pod.yml.erb"]["Pod"].first
   #     pod["spec"]["containers"].first["image"] = "hello-world:thisImageIsBad"
   #   end
-  def deploy_fixtures(set, subset: nil, **args) # extra args are passed through to deploy_dir_without_profiling
+  def deploy_fixtures(set, subset: nil, **args, &block) # extra args are passed through to deploy_dir_without_profiling
+    success, _ = deploy_fixtures_with_error(set, subset: subset, **args, &block)
```

Comment:
shouldn't we make sure that `_` is `nil`, so:

```
success, error = deploy_fixtures_with_error...
assert_equal error, nil
success
```

Reply:
Turns out several (20) tests break with this change.

Reply:
I think we should change that in the future, as you mentioned. And I am fine with not doing this it in this PR.

Reply:
This helper doesn't make any assumptions about the desired result; most (all?) tests that use `deploy_fixtures` should be consuming the return value with either `assert_deploy_failure` or `assert_deploy_success` (those helpers give you nice failure output when the result is incorrect). The 20 tests that fail are probably the ones where the deploy is  _expected_ to fail.